### PR TITLE
Add browser code, vex red, vex blue themes

### PR DESCRIPTION
**Pull request by Axorax**
#### Before you create your license page

Hosting costs $7 a month. I need help, so please donate a few months to keep this service going.

I've already had support to run the domain until 2032, but the hosting needs help too.

**[Please donate and keep this project running](https://www.paypal.me/rem)**

### CODE REVIEW

**Review by Cplus360 — APPROVED** [review]
Nice

**Comment by Axorax** [comment]
Hey! Can you merge the PR?

File: themes/vex-red.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    max-width: calc(100% - 2rem);
+    width: 60rem;
+    background: #fe3d3d;
+    color: #fff;
+    outline: 1px solid #ff9292;
+    padding: 1.5rem;
+}
+
+article {
+}
```
Comment (Richienb) [inline themes/vex-red.css:29]: Extraneous

File: themes/vex-blue.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    max-width: calc(100% - 2rem);
+    width: 60rem;
+    background: #5252ff;
+    color: #fff;
+    outline: 1px solid #9797ff;
+    padding: 1.5rem;
+}
+
+article {
+}
```
Comment (Richienb) [inline themes/vex-blue.css:29]: Extraneous

File: themes/vex-blue.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    text-align: center;
+}
+
+a {
+    color: #fff;
+    text-decoration: underline;
+}
+
+p:nth-child(3n) {
+    margin-bottom: 2.5rem!important;
```
Comment (Richienb) [inline themes/vex-blue.css:45]:
```suggestion
    margin-bottom: 2.5rem !important;
```

File: themes/vex-red.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    text-align: center;
+}
+
+a {
+    color: #fff;
+    text-decoration: underline;
+}
+
+p:nth-child(3n) {
+    margin-bottom: 2.5rem!important;
```
Comment (Richienb) [inline themes/vex-red.css:45]:
```suggestion
    margin-bottom: 2.5rem !important;
```

File: themes/browser-code.css [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+a {
+    text-decoration: none;
+    color: #fff;
+}
+
+html {
+    font-family: monospace;
+    background: #121212;
+    color: #FFF;
+    font-size: 13px!important;
```
Comment (Richienb) [inline themes/browser-code.css:17]:
```suggestion
    font-size: 13px !important;
```

**Review by Richienb — CHANGES_REQUESTED** [review]

**Pull request merged**